.watchmanconfig

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Sorry to commit this but jest uses watchman and watchman on my computer for some reason refuses to run without one outside of fb folders :(